src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
`ballColor` should be private because there is no need for any other class to access this field directly. All interaction with this field should be done via getter and setter - `getBallColor()` and `setBallColor()`.

Also, `ballColor` should be of type `core.basesyntax.Colors`, this secures the code from someone setting color to a value other than one of the possible values provided by the `Colors` enum.
This change will result in type change of all parts of the code using this field.